qoffer-domain/src/main/groovy/life/qbic/portal/portlet/customers/search/SearchCustomer.groovy

```diff
@@ -24,17 +24,15 @@ class SearchCustomer implements SearchCustomerInput{
 
     @Override
     void searchCustomer(String firstName, String lastName) {
-
         try {
             List<Customer> foundCustomer = dataSource.findCustomer(firstName, lastName)
-            int numberOfFoundCustomers = foundCustomer.size()
-            output.successNotification("Found $numberOfFoundCustomers customers matching $firstName $lastName")
-        } catch (DatabaseQueryException ignored) {
-            output.failNotification("Could not find a customer matching $firstName $lastName")
+            if (foundCustomer.isEmpty()) {
+                output.failNotification("Could not find a customer for $firstName $lastName")
+            } else {
+                output.successNotification(foundCustomer)
```

Comment:
Do we want to have the customer object list printed as success notification? I would prefer a list of matches or just a statment of how many customers have been found or so.

Reply:
No, the use case specification explicitly states, that a list of matching customers is returned from the use case.

Reply:
please see the SRS for reference, UC 4.2.6